### ISSUE TITLE
Missing `wasm-opt` when build the wasm package

### ISSUE THREAD

**Issue by FirelightFlagboy** [Scille/parsec-cloud#4240]
For example when build the libparsec web lib in the CI (during `web tests -> test e2e`) https://github.com/Scille/parsec-cloud/actions/runs/4434681006/jobs/7780911345

We have this error (I can replicate this error on my side :smile:).

```
Error: failed to execute `wasm-opt`: exited with signal: 11 (core dumped)
  full command: "/home/runner/.cache/.wasm-pack/wasm-opt-4d7a65327e9363b7/wasm-opt" "/home/runner/work/parsec-cloud/parsec-cloud/oxidation/bindings/web/pkg/index_bg.wasm" "-o" "/home/runner/work/parsec-cloud/parsec-cloud/oxidation/bindings/web/pkg/index_bg.wasm-opt.wasm" "-O"
To disable `wasm-opt`, add `wasm-opt = false` to your package metadata in your `Cargo.toml`.
```

To resolve this, either we install `wasm-opt` or we disable it.

**Comment by FirelightFlagboy** [comment]
I've added more context

**Comment by FirelightFlagboy** [comment]
I have taken a more in depth view of this issue.

The problem is on the `wasm-pack` side, it should install `wasm-opt` on it's own.

But with the current version we use `10.3` It fail to do do.

`wasm-pack` have release the version `11.0` but it segfault on my side If I install it via `npm`.

But I've manage to make it work if I install via `cargo install wasm-pack --no-default-features`.

see: <https://github.com/rustwasm/wasm-pack/issues/1186#issuecomment-1374814605>

**Comment by FirelightFlagboy** [comment]
I've tested if the issue is still present on e2b3cbdf2, and good news :)

That's resolved with `wasm-pack@0.11`
